hazelcast/src/main/java/com/hazelcast/internal/management/dto/ClientEndPointDTO.java

```diff
@@ -70,11 +70,11 @@ public void fromJson(JsonObject json) {
         address = getString(json, "address");
         clientType = getString(json, "clientType");
         name = getString(json, "name");
-        JsonObject attrObject = JsonUtil.getObject(json, "attributes");
-        attributes = new HashMap<String, String>();
-        for (JsonObject.Member member : attrObject) {
-            String value = member.getValue().asString();
-            attributes.put(member.getName(), value);
+        JsonObject labelsObject = JsonUtil.getObject(json, "labels");
+        labels = new HashSet<String>();
+        for (JsonObject.Member labelMember : labelsObject) {
```

Comment:
Most likely, deserialization should be updated too (see comment about duplicate keys in JSON).

Reply:
Corrected.